raiden/tests/integration/long_running/test_integration_events.py

```diff
@@ -551,7 +550,7 @@ def test_secret_revealed_on_chain(
         token_address=channel_state2_1.token_address,
         token_network_identifier=token_network_identifier,
         balance_proof=channel_state2_1.partner_state.balance_proof,
-        triggered_by_block_hash=make_block_hash(),
+        triggered_by_block_hash=app0.raiden.chain.block_hash(),
```

Comment:
You calling an attribute here.

Reply:
HAH! Thanks. I was debugging just this right now ^_^. But please take a look at the other smaller PR I pointed out in the chat first as I want to rebase this on top of this.

Reply:
@rakanalh hmm no this was wrong. I was completely absented minded when I resolved this conflict. This is correct as a function call here.

Reply:
Oh i see,,, you're calling the blockchain function